proto/mixer.proto

```diff
@@ -589,6 +589,20 @@ message InterestingPlaceAspect {
   repeated Aspect aspects = 1;
 }
 
+// Request for GetStat service.
+message GetStatValueRequest {
+  // dcid of the place.
+  string place = 1;
+  // dcid of the stat var.
+  string stat_var = 2;
+  // (optional) date of the stat.
+  string date = 3;
+}
+
+message GetStatValueResponse {
+  double value = 1;
```

Comment:
i know we only want the value back in the py api, but should we add other parameters for the value in the return object that would be useful? e.g. date the value was derived from?